filebeat/docs/reload-configuration.asciidoc

```diff
@@ -1,39 +1,17 @@
 [[filebeat-configuration-reloading]]
-== Reload the configuration dynamically
+== Load external configuration files
 
-beta[]
-
-You can configure Filebeat to dynamically reload configuration files when there
-are changes. This feature is available for prospector and module configuration
-files only.
-
-To configure this feature, you specify a path
-(https://golang.org/pkg/path/filepath/#Glob[Glob]) to watch for configuration
-changes. When the files found by the Glob change, new prospectors and/or
-modules are started and stopped according to changes in the configuration files.
-
-This feature is especially useful in container environments where one container
-is used to tail logs for services running in other containers on the same host.
-
-To enable dynamic config reloading, you specify the following options under
+Filebeat can load external configuration files for prospectors and modules,
+which allows you to separate your configuration in multiple smaller
+configuration files. To use this functionality, use the `path` option from the
```

Comment:
I wonder if we really need to mention the options here since we describe them more completely later. Might actually be a bit confusing. 

Maybe just add links to the sections that follow and say:

To use this functionality, specify the `path` option when you configure the <<reload-prospector-config,prospector>> or <<reload-module-config,module>>.

BTW, I would probably rename the anchors to load-prospector-config and load-module-config.